rules/common.smk

```diff
@@ -34,12 +34,14 @@ class VpipeConfig(object):
             'fastq_suffix': __RECORD__(value='', type=str),
             'trim_percent_cutoff': __RECORD__(value=0.8, type=float),
             'reference': __RECORD__(value='references/HXB2.fasta', type=str),
+            'gff_directory': __RECORD__(value='', type=str),
```

Comment:
Given that these files describe information about the virus genome (i.e.: about the reference), do you think it would be a good idea to move them in a subdirectory there ? (i.e.: `references/gff/`)

It would also make `./init_project.sh` still work as is.